services/web-app/package.json

```diff
@@ -29,11 +30,19 @@
     "cache-manager": "^3.4.4",
     "cache-manager-fs-hash": "^1.0.0",
     "clsx": "^1.1.1",
+    "connect-sqlite3": "^0.9.12",
```

Comment:
https://www.npmjs.com/package/connect-sqlite3

This dependency does not have a license. We need to only use open-source dependencies with permissible licenses like MIT and Apache-2.0. Does this package actually have a license anywhere? If not, is there an alternative?

Reply:
updated to version 0.9.13, containing MIT license file